url_launcher 2.0.0 breaks flutter build

url_launcher 1.0.3 is followed by url_launcher 2.0.0, which is not compatible with flutter build at this point:

```
[   +3 ms] /b/build/slave/Linux/build/flutter/examples/flutter_gallery/android/gradlew -v
[ +533 ms] 
           ------------------------------------------------------------
           Gradle 3.3
           ------------------------------------------------------------
...           
           BUILD FAILED
           
           Total time: 1.383 secs
           
           
           FAILURE: Build failed with an exception.
           
           * Where:
           Build file '/b/build/slave/Linux/build/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher-2.0.0/android/build.gradle' line: 6
           
           * What went wrong:
           A problem occurred evaluating project ':url_launcher'.
           > Could not find method google() for arguments [] on repository container.
           
           * Try:
           Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
           
[  +15 ms] "flutter apk" took 2,279ms.
```

--- a/examples/flutter_gallery/pubspec.yaml
+++ b/examples/flutter_gallery/pubspec.yaml
@@ -3,11 +3,11 @@ dependencies:
   flutter:
     sdk: flutter
   collection: 1.14.3
-  device_info: 0.0.5
+  device_info: 0.1.0
   intl: 0.15.2
-  connectivity: 0.1.1
+  connectivity: 0.2.0
   string_scanner: 1.0.2
-  url_launcher: 1.0.3
+  url_launcher: 2.0.0
   cupertino_icons: 0.1.1
   video_player: 0.0.7
 

--- a/packages/flutter_tools/lib/src/commands/update_packages.dart
+++ b/packages/flutter_tools/lib/src/commands/update_packages.dart
@@ -30,18 +30,6 @@ const Map<String, String> _kManuallyPinnedDependencies = const <String, String>{
   // https://github.com/dart-lang/sdk/issues/31442
   // TODO(amirh): unpin this.
   'test': '0.12.26',
-  // Major version upgrade to next version(2.0.0) breaks flutter build:
-  // https://github.com/flutter/flutter/issues/13953
-  // TODO(aam): unpin this.
-  'url_launcher': '1.0.3',
-  // Version upgrade to next version(0.2.0) breaks flutter build:
-  // https://github.com/flutter/flutter/issues/13958
-  // TODO(aam): unpin this.
-  'connectivity' : '0.1.1',
-  // Version upgrade to next version(0.1.0) breaks flutter build:
-  // https://github.com/flutter/flutter/issues/13956
-  // TODO(aam): unpin this.
-  'device_info': '0.0.5',
   // Add pinned packages here.
 };
 

